apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -314,6 +338,25 @@ export default class SetupChecklist extends Component {
             </a>
             .{this.contactSupport()}
           </ValidationStep>
+          {experiments.isEnabled('flash-classic') &&
+            this.state.boardTypeDetected === BOARD_TYPE.CLASSIC && (
+              <ValidationStep
+                stepStatus={this.state[STATUS_BOARD_FIRMWARE]}
+                stepName="Board firmware up-to-date"
+              >
+                <div>
+                  This board is not using the most up-to-date version of the
```

Comment:
Temp text until we get final text from Ryan's design

Reply:
Localization? (once we get the final text..)

Reply:
I should probably localize now, so I don't forget to do it later :laughing: